esrally/mechanic/provisioner.py

```diff
@@ -184,7 +186,17 @@ def _provisioner_variables(self):
         plugin_variables = {}
         mandatory_plugins = []
         for installer in self.plugin_installers:
-            mandatory_plugins.append(installer.plugin_name)
+            # For Elasticsearch < 6.3 more specific plugin names are required for mandatory plugin check
+            # Details in: https://github.com/elastic/elasticsearch/pull/28710
+            try:
+                if ((versions.major_version(self.distribution_version) == 6 and
```

Comment:
Alternatively you could use `major, minor, _, _ = versions.components(self.distribution_version)`. Then there would also be no need for a new helper function `minor_version` in the `versions` module.

Reply:
Yes good suggestion! I felt like simplifying the code inside the `_provisioner_variables()` method with no need to assign temporary variables but there are ways around it and I agree let's not add another helper function.